cmd/ocm-backplane/cloud/console_test.go

```diff
@@ -57,7 +57,7 @@ var _ = Describe("Cloud console command", func() {
 		// Define fake AWS response
 		fakeAWSResp = &http.Response{
 			Body: MakeIoReader(
-				fmt.Sprintf(`{"proxy_uri":"proxy", "statusCode":200, "message":"msg", "ConsoleLink":"%s"}`, consoleAWSUrl),
+				fmt.Sprintf(`{"proxy_uri":"proxy", "message":"msg", "ConsoleLink":"%s"}`, consoleAWSUrl),
```

Comment:
Any plausible explanation for why `statusCode` was removed?

Reply:
Status code is already defined fakeAWSResponse as follows
https://github.com/openshift/backplane-cli/blob/3b96b78a52df8073ecb4becb8ffccdd7e17cbfad/cmd/ocm-backplane/cloud/console_test.go#L63

If it's defined via the massage-body, then it may not be able to update when it's testing for other status codes.
eg : status code 500 tests
https://github.com/openshift/backplane-cli/blob/3b96b78a52df8073ecb4becb8ffccdd7e17cbfad/cmd/ocm-backplane/cloud/console_test.go#L137